From 279ae77d41dafa878bf68b33fff551e4155b23c7 Mon Sep 17 00:00:00 2001
From: Tri Luong <tri.luong.te@renesas.com>
Date: Fri, 7 Jun 2019 09:55:27 +0700
Subject: [PATCH 213/248] arm64: dts: renesas: hihope-common: Define and enable
 Bluetooth pin.

This patch defines and enables pin for bluetooth device
used for RZ/GM board (R8A774A1 SoC).

Signed-off-by: Tri Luong <tri.luong.te@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-common.dtsi | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/hihope-common.dtsi b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
index a56d32e..37d6679 100644
--- a/arch/arm64/boot/dts/renesas/hihope-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
@@ -481,3 +481,12 @@
 
 	status = "okay";
 };
+
+&gpio3 {
+	bluetooth-en-gpio {
+		gpio-hog;
+		gpios = <13 GPIO_ACTIVE_HIGH>;
+		output-high;
+		line-name = "bluetooth-en-gpio";
+	};
+};
-- 
2.7.4

